doc/admin-guide/files/records.config.en.rst

```diff
@@ -1090,6 +1090,15 @@ mptcp
 
 .. ts:cv:: CONFIG proxy.config.http.default_buffer_water_mark INT 32768
 
+.. ts:cv:: CONFIG proxy.config.http.request_buffer_enabled INT 0
+   :overridable:
+
+   This is a configuration value that is overridable but not configurable. This is most likely an
```

Comment:
"but not configurable" is confusing to me. I read that as "you may not put this into records.config," which seems unlikely? Is it meant to be, overridable, but not reloadable?

Reply:
That is correct, if you put that in `records.config` you'll get an error of "no such variable". I sent email to the dev mailing list about this, as it seems quite wrong to me. I had to put something in to get the documentation build errors cleaned up, but I couldn't write a normal entry as that won't work.

Reply:
Wacky!!

Reply:
From master
```
[Jan 10 11:21:41.667] traffic_server WARNING: Unrecognized configuration value 'proxy.config.http.request_buffer_enabled'
```

Reply:
I think this is an old configuration, that should be removed. I looked, and the overridable variable (c.oride.request_buffer_enabled,) is never used. So pretty sure we nuked this config, but forgot to remove from docs and overridable?

Reply:
Gah, I missread, yeh, this is some whacky stuff.

Reply:
I think @zizhong put that in and said he would put up a PR Real Soon Now to fix it. I think we should commit this and have him update it in his PR.